itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/txn/compactor/CompactorOnTezTest.java

```diff
@@ -85,6 +86,8 @@ public void setup() throws Exception {
     hiveConf.setVar(HiveConf.ConfVars.METASTOREWAREHOUSE, TEST_WAREHOUSE_DIR);
     hiveConf.setVar(HiveConf.ConfVars.HIVEINPUTFORMAT, HiveInputFormat.class.getName());
     hiveConf.setVar(HiveConf.ConfVars.HIVEFETCHTASKCONVERSION, "none");
+    hiveConf.set(MetastoreConf.ConfVars.TXN_OPENTXN_TIMEOUT.getVarname(), "2000");
```

Comment:
We should use the code below to set MetastoreConf values to handle migration between HiveConf, MetastoreConf:
```
    MetastoreConf.setVar(hiveConf, MetastoreConf.ConfVars.TXN_OPENTXN_TIMEOUT, "2000");
```

This is not too relevant here since this is only test code, but I prefer to stick to the standard way everywhere, so the next contributor will learn correctly.